packages/arb-token-bridge-ui/src/util/__tests__/isDepositMode.test.ts

```diff
@@ -0,0 +1,129 @@
+import { registerCustomArbitrumNetwork } from '@arbitrum/sdk'
+
+import { isDepositMode } from '../isDepositMode'
+import { ChainId } from '../networks'
+import { orbitMainnets } from '../orbitChainsList'
+
+beforeAll(() => {
+  const xaiMainnetChainId = 660279
+
+  const xaiMainnet = orbitMainnets[xaiMainnetChainId]
+
+  if (!xaiMainnet) {
+    throw new Error(`Could not find Xai Mainnet in the Orbit chains list.`)
+  }
+
+  registerCustomArbitrumNetwork(xaiMainnet)
+
+  const rariMainnetChainId = 1380012617
+
+  const rariMainnet = orbitMainnets[rariMainnetChainId]
+
+  if (!rariMainnet) {
+    throw new Error(`Could not find Rari Mainnet in the Orbit chains list.`)
+  }
+
+  registerCustomArbitrumNetwork(rariMainnet)
+})
+
+describe('isDepositMode', () => {
+  it('should return true for L1 source chain and L2 destination chain', () => {
+    const result1 = isDepositMode({
+      sourceChainId: ChainId.Ethereum,
+      destinationChainId: ChainId.ArbitrumOne
+    })
+    expect(result1).toBe(true)
+
+    const result2 = isDepositMode({
+      sourceChainId: ChainId.Ethereum,
+      destinationChainId: ChainId.ArbitrumNova
+    })
+
+    expect(result2).toBe(true)
+  })
+  it('should return true for L2 source chain and L3 destination chain', () => {
+    const result1 = isDepositMode({
+      sourceChainId: ChainId.ArbitrumOne,
+      destinationChainId: 660279 // Xai
```

Comment:
Not related to this PR, but it would be nice to have enum for chain ids somewhere

Reply:
perhaps we can generate enums from the orbit chains data json, but i think the original idea was to only keep core chain's ids as enum